examples/all-clusters-app/esp32/main/main.cpp

```diff
@@ -705,146 +179,9 @@ extern "C" void app_main()
         return;
     }
 
-    statusLED1.Init(STATUS_LED_GPIO_NUM);
-    // Our second LED doesn't map to any physical LEDs so far, just to virtual
-    // "LED"s on devices with screens.
-    statusLED2.Init(GPIO_NUM_MAX);
-    bluetoothLED.Init();
-    wifiLED.Init();
-    pairingWindowLED.Init();
-
-    chip::DeviceLayer::PlatformMgr().ScheduleWork(InitServer, reinterpret_cast<intptr_t>(nullptr));
-
-    // Print QR Code URL
-    PrintOnboardingCodes(chip::RendezvousInformationFlags(CONFIG_RENDEZVOUS_MODE));
-
     InitOTARequestor();
```

Comment:
InitOTARequestor before any of the CHIP APIs are called or before the InitServer is done?

Reply:
I didn't change the location of InitOTARequestor。